frontend/src/pages/customer/OrderHistory.tsx

```diff
@@ -0,0 +1,46 @@
+import { useForm } from "react-hook-form";
+import { useQuery } from "@tanstack/react-query";
+import { listEvents } from "../../axios/event/event";
+import { Grid } from "@mui/material";
+import OrderCard from "../../components/event/card/OrderCard";
+import TabBar from "../../components/event/TabBar/TabBar";
+
+const OrderHistory = () => {
+    const formState = useForm();
+    const { watch } = formState;
+    const [searchVal, categoryVal] = watch(["search", "category"]);
+
+    const { data: events } = useQuery({
+        queryKey: ["events", searchVal, categoryVal],
+        queryFn: () => listEvents(false),
+    });
+
+const labels = [
+    "Purchase Requests",
+    "Order History"
+];
+
+    return (
+        <>
+        <TabBar labels={labels} displayBuyTicketButton={false}></TabBar>
```

Comment:
For boolean type props, if you don't mention `displayBuyTicketButton` here, React knows that `displayBuyTicketButton` is `false`, This means

`<TabBar displayBuyTicketButton={false} />`

is the same as

`<TabBar />